src/utils/perfherder/chartJsFormatter.js

```diff
@@ -0,0 +1,60 @@
+const dataToChartJSformat = data =>
+  data.map(({ datetime, value }) => ({
+    x: datetime,
+    y: value,
+  }));
+
+const chartJsOptions = (reverse, scaleLabel) => ({
+  scales: {
+    xAxes: [
+      {
+        type: 'time',
+        time: {
+          displayFormats: {
+            hour: 'MMM D',
+          },
+        },
+      },
+    ],
+    yAxes: [
+      {
+        ticks: {
+          beginAtZero: true,
+          reverse,
+        },
+        scaleLabel: {
+          display: true,
+          labelString: scaleLabel,
+        },
+      },
+    ],
+  },
+});
+
+const generateChartJsOptions = (meta) => {
+  const higherIsBetter = (meta.lower_is_better === false);
+  const reversed = higherIsBetter;
+  const yLabel = higherIsBetter ? 'Score' : 'Execution time (ms)';
+  return chartJsOptions(reversed, yLabel);
```

Comment:
It doesn't seem like you need the reversed const - couldn't you pass in higherIsBetter const instead?